integration/clean/main.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent/integration/clean/ami"
+	"github.com/aws/amazon-cloudwatch-agent/integration/clean/dedicated_host"
+	"github.com/aws/amazon-cloudwatch-agent/integration/clean/ssm"
+)
+
+const (
+	defaultDaysToKeep = 5
```

Comment:
This default value does not line up with the default value in the GitHub action. I'm indifferent as to which one we use, and ultimately the action value will overwrite this every time, but I'd rather have them match.

Reply:
Agree. It would better to consistence with the Github Workflow. Changed it in the newest commit.